src/main/resources/db/migration/V11.01__undo_adding_records_to_accounts.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM account WHERE email = 'ivan.t.tsenov@gmail.com';
```

Comment:
No need of new migration file, just put these deletes in a single migration fail with other code